src/test/java/com/sports/server/command/game/application/GameServiceTest.java

```diff
@@ -136,5 +138,51 @@ private void assertLineupPlayers(List<GameTeam> gameTeams) {
         }
     }
 
+    @Nested
+    @DisplayName("게임을 수정할 때")
+    class updateGameTest {
+
+        private GameRequestDto.Update updateDto;
+        private Long leagueId;
+        private Long gameId;
+        private Member manager;
+
+        @BeforeEach
+        void setUp() {
+            LocalDateTime fixedLocalDateTime = LocalDateTime.of(2024, 9, 11, 12, 0, 0);
+            updateDto = new GameRequestDto.Update(nameOfGame, "8강", "후반전", "PLAYING",
+                    fixedLocalDateTime, "videoId");
+            leagueId = 1L;
+            gameId = 1L;
+            manager = entityUtils.getEntity(1L, Member.class);
+        }
+
+        @Test
+        void 정상적으로_게임이_수정된다() {
+            // when
+            gameService.updateGame(leagueId, gameId, updateDto, manager);
+
+            // then
+            Game game = entityUtils.getEntity(gameId, Game.class);
+            assertThat(game.getGameQuarter()).isEqualTo(updateDto.quarter());
+            assertThat(game.getRound()).isEqualTo(Round.from(updateDto.round()));
+            assertThat(game.getName()).isEqualTo(updateDto.name());
+            assertThat(game.getStartTime()).isEqualTo(updateDto.startTime());
+            assertThat(game.getState()).isEqualTo(GameState.from(updateDto.state()));
+            assertThat(game.getVideoId()).isEqualTo(updateDto.videoId());
```

Comment:
여기도 assertAll()을 사용하면 좋겠네요.
assertAll()을 사용하면 어떤 Assertion이 통과 안되는지 한번에 볼 수 있어요.
```suggestion
     assertAll(
            () -> assertThat(game.getGameQuarter()).isEqualTo(updateDto.quarter()),
            () -> assertThat(game.getRound()).isEqualTo(Round.from(updateDto.round())),
            () -> assertThat(game.getName()).isEqualTo(updateDto.name()),
            () -> assertThat(game.getStartTime()).isEqualTo(updateDto.startTime()),
            () -> assertThat(game.getState()).isEqualTo(GameState.from(updateDto.state())),
            () -> assertThat(game.getVideoId()).isEqualTo(updateDto.videoId())
        );
```